doc/about.rst

```diff
@@ -69,11 +76,6 @@ The following people have been active contributors in the past, but are no longe
 - Virgile Fritsch
 - Wei Li
```

Comment:
In my opinion, we should really kill this author list, because it is never up to date, and point to the list generated by github.

However, this could go in another PR.